### ISSUE TITLE
Improve accessibility score by giving buttons accessible names on document view page

### ISSUE THREAD

**Issue by FlowingCloudRTL** [CMU-313/Teedy#213]
Currently, after uploading an image to the document page, the accessibility score reported by Lighthouse is 76. 
In addition to the document page's existing issues with missing accessible names at "Add a document" dropdown, folder and tag buttons, by viewing a document, a new button next to the file name also does not have an accessible name to it, causing screen reader users to have difficulty identifying these buttons.

**Comment by FlowingCloudRTL** [comment]
I would like to work on this please!
